components/developer/rust/Makefile

```diff
@@ -146,7 +146,6 @@ REQUIRED_PACKAGES += $(GCC_RUNTIME_PKG)
 REQUIRED_PACKAGES += $(GXX_RUNTIME_PKG)
```

Comment:
Is developer/gcc-13 still needed?

Reply:
There are far more things in the `Makefile` that are not needed.  For example the touch of `META.depend-runtime.res` or `CLANG_VERSION` setting.